### DISCUSSION TITLE
DataTable focus cell programmatically

### DISCUSSION THREAD

**Discussion by kanalje** [primefaces/community#725] [PrimeVue]
I have a DataTable with editMode="cell". Is there a way to programmatically move focus to a given cell?

**Comment by tugcekucukoglu** [comment]
Since DataTable has its API for editing modes, usage in cell edit mode is more restrictive. Row edit mode is more flexible because it has additional row edit columns. We can still pass cell edit events to the slot. Exactly what kind of programmatic edit are you thinking of?

Row edit templating example: https://stackblitz.com/edit/v4dxa4?file=src%2FApp.vue
